addons/l10n_ar/models/account_chart_template.py

```diff
@@ -0,0 +1,46 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api, fields
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    # MOVER A LOGICA DE def _get_fp_vals(self, company, position):
+    @api.multi
+    def generate_fiscal_position(
+            self, tax_template_ref, acc_template_ref, company):
+        """ If chart is argentina, then we add l10n_ar_afip_code to
+        fiscal positions. We also add other data to add fiscal positions
+        automatically
+        """
+        res = super(AccountChartTemplate, self).generate_fiscal_position(
+            tax_template_ref, acc_template_ref, company)
+        if company.country_id.code != 'AR':
```

Comment:
Why not with xml_id?


```suggestion
        if company.country_id == self.env.ref('base.ar'):
```

Reply:
IMHO I think that is better to use 'AR instead on the self.env.ref(). 

This because the ref() method and inner calls to other methods do several calls to browse() , search() and read(). Do we actually need to add this cost reading the database when we already now the value we want to compare?

Reply:
A requirement in the Mexican Localization was always used the xml_id, this to ensure that is the record that We need. All the localization use the `self.env.ref()`,

Reply:
Not in all places.  In the reports e.g. country_id.code is quite common.

Reply:
I understand. I was not aware about the requirement, thank you for the feedback.